workflows/microbiome-analysis/README.md

```diff
@@ -0,0 +1,60 @@
+# Microbiome Analysis workflows for Nanopore datasets - collection version
+
+The following workflows can be used directly for any nanopore metagenomics data analysis, pathogen detection and tracking purposes. The workflows can be also adapted to any other sequencing technique. 
+
+To learn more about all workflows and try them with real datasets, please check our tutorial on the Galaxy Training Network [GTN](https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.html)
+
+## Pre-Processing
```

Comment:
Remove pathogen detection from folder name in preprocessing and taxonomy profiling